Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/ui/selection/TestRegistrationSelectionFragment.kt

```diff
@@ -1,6 +1,38 @@
 package de.rki.coronawarnapp.familytest.ui.selection
 
+import android.os.Bundle
+import android.view.View
 import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.navArgs
 import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentTestRegistrationSelectionBinding
+import de.rki.coronawarnapp.util.ui.doNavigate
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
 
-class TestRegistrationSelectionFragment : Fragment(R.layout.fragment_test_registration_selection)
+class TestRegistrationSelectionFragment : Fragment(R.layout.fragment_test_registration_selection) {
+
+    private val navArgs by navArgs<TestRegistrationSelectionFragmentArgs>()
+    private val binding: FragmentTestRegistrationSelectionBinding by viewBinding()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.toolbar.setNavigationOnClickListener { popBackStack() }
+        binding.person.setOnClickListener {
+            doNavigate(
+                TestRegistrationSelectionFragmentDirections
+                    .actionTestRegistrationSelectionFragmentToSubmissionConsentFragment(
```

Comment:
that is not always the case , if there a test of the same type , you need to open the duplicate warning screen , see Figma